packages/perseus-core/src/__tests__/version.test.ts

```diff
@@ -0,0 +1,16 @@
+describe("version", () => {
+    beforeEach(() => {
+        delete globalThis.__perseus_debug__;
+    });
+
+    it("adds the perseus-core version to globalThis", async () => {
+        await import("../version");
+
+        expect(globalThis).toHaveProperty("__perseus_debug__");
+        expect(globalThis.__perseus_debug__).toMatchInlineSnapshot(`
+            {
+              "@khanacademy/perseus-core": "__lib_version__",
+            }
+        `);
+    });
```

Comment:
I'm not 100% sure if this is helpful.  If I revert the change in `version.ts` it still passes so this doesn't actually protect against regression here.  

Any thoughts?

Reply:
Modifying `globalThis` in a test makes me nervous. The fact that the test doesn't catch regressions makes me inclined to remove it.

I'd rather lean on a linter or something to check invariants like "a package should never import its own code from `node_modules`" (if that is in fact what we want to enforce... I'm not sure what the rule should be).

Reply:
You've convinced me. I'll remove the test and land this.  I'll also do a small amount of research if this would be relatively easy to lint for (maybe with ESLint's dynamic rules).